app/templates/home.html

```diff
@@ -12,11 +12,9 @@
     <h1 class="site-title">BeeStation, For Beginners</h1>
     <p class="text-indent">It appears you have stumbled into the cozy corner of the BeeStation website. It has the rules for the server, a server statistics page, a page to review server bans, a forum, and links to many more places you should visit!</p>
 
-    <p class="text-indent">BeeStation is a Space Station 13 server, hosted by Crossedfall, which was created in March 2018. We focus greatly on being newbie friendly. We are a Low RP server. Our codebase is /tg/, but we are constantly striving to change up our code and add in new features. We love to put a touch of fun into every round to keep things interesting.</p>
+    <p class="text-indent">BeeStation is a Space Station 13 server, hosted by Crossedfall, which was created in March 2018.  We focus on being being a newbie friendly community, with LRP (Golden) and MRP (Sage) servers. Our codebase is built off of /tg/'s foundations, combining our own unique features with those found on other servers.</p>
```

Comment:
```suggestion
    <p class="text-indent">BeeStation is a Space Station 13 server, hosted by Crossedfall, which was created in March 2018.  We focus on being a newbie friendly community, with LRP (Golden) and MRP (Sage) servers. Our codebase is built off of /tg/'s foundations, combining our own unique features with those found on other servers.</p>
```

I made an oopsie in my corrections. Added an extra `being`. Sorry about that